package.json

```diff
@@ -10,14 +10,10 @@
   },
   "devDependencies": {
     "babel-core": "^6.26.0",
-    "babel-jest": "^22.0.6",
+    "babel-jest": "^23.6.0",
     "babel-preset-es2015": "^6.24.1",
     "eslint": "^4.14.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.5.1",
-    "jest": "^22.0.6",
-    "lerna": "^2.7.1",
-    "regenerator-runtime": "^0.11.1"
+    "jest": "^23.6.0",
+    "lerna": "^3.6.0"
```

Comment:
Note the major version bump of Lerna. Hoping that fixes some of the bad errors messages we saw when publishing got messed up.